docs/.vuepress/components/ListCard.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+  <div class="list-new max-w-768px">
+    <template v-for="(item, i) in items">
+      <router-link
+        v-if="item.isInternal"
```

Comment:
I don't see `isInternal` defined/used anywhere. In any case, instead of passing in a prop to tell if a link is internal, we can use a helper function for this - see `isExternal` in util.js. You can see how we also use that in NavLink.vue (`v-if="!isExternal(link)"`).

Reply:
Ah! Thank you!